site/schemas/create-consequence-stops.schema.ts

```diff
@@ -0,0 +1,130 @@
+import { Severity } from "@create-disruptions-data/shared-ts/enums";
+import { z } from "zod";
+import { setZodDefaultError, zodTimeInMinutes } from "../utils";
+
+export const stopsImpactedSchema = z.object({
+    id: z.number({
+        required_error: "Id is required",
```

Comment:
I wouldn't bother overriding the errors if they're not being displayed to the user, zod will return an error with more detail by default

Reply:
done